src/main/features/claim/routes/new-features-consent.ts

```diff
@@ -0,0 +1,42 @@
+import * as express from 'express'
+import { Paths as ClaimPaths } from 'claim/paths'
+import { Form } from 'forms/form'
+import { FeatureConsentResponse } from 'forms/models/featureConsentResponse'
+import { NewFeaturesConsentGuard } from 'claim/guards/newFeaturesConsentGuard'
+import { FormValidator } from 'forms/validation/formValidator'
+import { ErrorHandling } from 'shared/errorHandling'
+import { ClaimStoreClient } from 'claims/claimStoreClient'
+import { YesNoOption } from 'models/yesNoOption'
+import { User } from 'idam/user'
+
+const claimStoreClient = new ClaimStoreClient()
+
+function renderView (form: Form<FeatureConsentResponse>, res: express.Response) {
+  res.render(ClaimPaths.newFeaturesConsent.associatedView, { form: form })
+}
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(ClaimPaths.newFeaturesConsent.uri,
+    NewFeaturesConsentGuard.requestHandler, (req: express.Request, res: express.Response) => {
```

Comment:
Since `NewFeaturesConsentGuard.requestHandler` will be only applied once and possible reuse is minimal I would recommend to store that guard locally in this file as we do in other routes.